src/gnatss/solver/utilities.py

```diff
@@ -387,15 +387,18 @@ def prepare_and_solve(
     # Store original xyz
     original_positions = transponders_xyz.copy()
 
+    # Store number of transponders
+    num_transponders = len(transponders)
+
     typer.echo("Preparing data inputs...")
-    data_inputs = get_data_inputs(all_observations)
+    reduced_observations = prefilter_replies(all_observations, num_transponders)
```

Comment:
Maybe add a message output here for this process so that you're aware:

```python
typer.echo("Prefiltering replies of <3 ...")
```

Something like that. This way, you're always aware that current software is doing that rather than just have this hidden.

Reply:
Done.